projects/chisel.yml

```diff
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
 # Did the project put links to CHIPS in the header or footer of its website? If no website, still answer "yes" as declaration of will to do so.
-chips_header_footer_text_on_website: yes
+chips_header_footer_text_on_website: not yet
 # Did the project transfer any registered trademarks and domains to CHIPS? If none are present, still answer "yes" as declaration of intent if that should change.
-trademarks_and_domains_transferred_to_lf: not yet
+trademarks_and_domains_transferred_to_lf: yes
```

Comment:
I don't think we have any trademarks, and we have transferred chisel-lang.org over.

Reply:
Excellent.